src/rust-crypto/backup.ts

```diff
@@ -20,11 +20,14 @@ import * as RustSdkCryptoJs from "@matrix-org/matrix-sdk-crypto-wasm";
 import { BackupTrustInfo, Curve25519AuthData, KeyBackupCheck, KeyBackupInfo } from "../crypto-api/keybackup";
 import { logger } from "../logger";
 import { ClientPrefix, IHttpOpts, MatrixError, MatrixHttpApi, Method } from "../http-api";
-import { CryptoEvent } from "../crypto";
+import { CryptoEvent, IMegolmSessionData } from "../crypto";
 import { TypedEventEmitter } from "../models/typed-event-emitter";
 import { encodeUri } from "../utils";
 import { OutgoingRequestProcessor } from "./OutgoingRequestProcessor";
 import { sleep } from "../utils";
+import { BackupDecryptor } from "../common-crypto/CryptoBackend";
+import { IEncryptedPayload } from "../crypto/aes";
+import { IKeyBackupSession, Curve25519SessionData } from "../crypto/keybackup";
```

Comment:
I'd love it if we could move utilities like this out of `crypto` while we are working on them (I basically see `src/crypto` as "Legacy Crypto"), but we can do that later if you prefer

Reply:
Moved to crypto-api/keybackup.ts like other similar utilities. Added re-export for backward compatibility.
I used the same convention for the other utilities where the `I` prefix is removed for new names.
Updated the new files to use the new import.